packages/security/src/deviceTrust/checks/RootedCheck.ts

```diff
@@ -0,0 +1,22 @@
+import { SecurityCheck } from "../SecurityCheck";
+import { SecurityCheckResult } from "../SecurityCheckResult";
+
+declare var IRoot: any;
+
+export class RootedCheck implements SecurityCheck {
+  public readonly name = "Rooted Check";
+
+  public test(): Promise<SecurityCheckResult> {
+    return new Promise((resolve, reject) => {
+      if (!IRoot) {
+        reject(new Error("Could not find plugin IRoot."));
+        return;
+      }
+
+      IRoot.isRooted((rooted: number) => {
+        const result = new SecurityCheckResult(this.name, !!rooted);
```

Comment:
That's strange in TS. You usually do something like:

```
const result: SecurityCheckResult = { name, rooted }
```

Reply:
That makes sense, will address this